src/alchemlyb/visualisation/convergence.py

```diff
@@ -33,7 +33,7 @@ def plot_convergence(forward, forward_error, backward, backward_error,
     `Alchemical Analysis <https://github.com/MobleyLab/alchemical-analysis>`_.
 
     The units variable is for labelling only. Changing it doesn't change the
-    unit of the underlying variable, which is in the unit of :math:`kT`.
+    unit of the underlying variable.
```

Comment:
Does `plot_convergence()`  not do unit conversion, even though the other plotting functions do? 

Can you change it, not the least the CHANGES says that this is now working for all visualization functions.

Reply:
Or remove `units` and take the label from the df `attr` — but having the same behavior as the other plotting functions would be much better and more consistent.

Reply:
There is currently no alchemlyb function that will generate the input for plot_convergence.
The user needs to generate the input file, which could be a simple python list.